docs/organization/integrations/index.mdx

```diff
@@ -127,3 +127,11 @@ For more details, see the [full Integration Platform documentation](/organizatio
 | [AWS Lambda - Python (Manual)](/platforms/python/integrations/aws-lambda/)            | X               | X                          |
 | [AWS Lambda - Node (Automatic)](/organization/integrations/cloud-monitoring/aws-lambda/)   | X               | X                          |
 | [AWS Lambda - Node (Manual)](/platforms/javascript/guides/aws-lambda/)                      | X               | X                          |
+
+
+## Toggleable Features
+Some integrations have features that can be enabled or disabled through a toggle at the integration level. The toggle happens instantly across the feature, and defaults are dependent on the feature, per integration.
```

Comment:
```suggestion
Some integrations have features that can be enabled or disabled through a toggle at the integration level. Toggling features on/off takes effect instantly across the integration. Defaults are dependent on the feature, per integration.
```
Not sure what you mean by "the toggle happens instantly." Not sure if you meant the above or maybe you mean "The toggle appears automatically for available features for each integration"? but that's also kinda assumed I think.

Reply:
Yeah here I meant it takes effect instantly instead of there being a delay.

Reply:
Used your suggestion, thank you!